README.md

```diff
@@ -1,7 +1,57 @@
-# eq-fault-geom
-python tools for manipulating fault geometries 
+## eq-fault-geom
 
-# Proposed initial layout of Python package.
+Python tools for manipulating fault geometries.
+
+### Getting started
+
+##### 1. Clone the repository
+
+````
+git clone https://github.com/GNS-Science/eq-fault-geom.git
+````
+
+##### 2. Setup a Python virtulenv
+
+pre-requisites:
+
+* python >=Python 3.7
+
+* python pip3
+
+* python virtualenv
+
+##### 3. Create and activate Python virtualenv
+
+````
+mkdir eq-fault-geom/ENV
```

Comment:
IN hindsight these two lines are best left out. I will remove them from the other project too.